chapter_appendix-tools-for-deep-learning/jupyter.md

```diff
@@ -115,12 +115,25 @@ The above is the address of the remote server `myserver`. Then we can use http:/
 
 We can use the `ExecuteTime` plugin to time the execution of each code cell in a Jupyter Notebook. Use the following commands to install the plugin:
 
-```
+```sh
 pip install jupyter_contrib_nbextensions
 jupyter contrib nbextension install --user
 jupyter nbextension enable execute_time/ExecuteTime
 ```
 
+### Hinter (without using `tab`)
+
+We can use the `Hinterland` plugin to give a hint without using tab while coding. Use the following commands to install the plugin:
+
+```sh
+pip install jupyter_contrib_nbextensions_configurator
+jupyter nbextensions_configurator enable --user
+```
+
+And we need to check.![What we need to check](../img/jupyter_check.png)
```

Comment:
`![What we need to check](../img/jupyter_check.png)` needs to sit in a whole line.

Reply:
also please reduce the png file size, e.g. reducing the resolution.

Reply:
Got it!